src/components/Home.js

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector} from 'react-redux';
+import { showCountries } from '../redux/home/home';
+import { useNavigate } from 'react-router-dom';
+import '../Home.css';
+
+const Home = () => {
+
+  const southern = [
+    'Angola',
+    'Botswana',
+    'Lesotho',
+    'Mozambique',
+    'Namibia',
+    'South Africa',
+    'Swaziland',
+    'Zambia',
+    'Zimbabwe',
+  ];
+
+  const state = useSelector((state) => state.country.filter((country) => southern.includes(country.name)));
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  console.log(state);
```

Comment:
- [ ] **Optional:** Kindly remove all `console.log`s from your code before deploying or committing. It is considered professional and part of best practice.
_**Hint**_  Do a global search for `log`s in your code. 

![image](https://user-images.githubusercontent.com/27709832/169064857-8f3ba4ec-055a-44f2-b43e-279c97a88436.png)
  > global search icon in vscode

You could use the `logger` middleware to see the state of your app in development, alternatively.